executor/src/witgen/jit/witgen_inference.rs

```diff
@@ -456,11 +459,27 @@ impl<'a, T: FieldElement, FixedEval: FixedEvaluator<T>> WitgenInference<'a, T, F
         true
     }
 
+    /// Record a variable as known. Return true if it was not known before.
+    fn record_known(&mut self, variable: Variable) -> bool {
+        // We do not record fixed columns as known.
```

Comment:
A consequence of this is that it is not returned by `known_variables()`, and a consequence of that is that we'll never branch on fixed column values. I don't see where this would be useful, but it's not the most consistent behavior.

Reply:
Would it hurt to initialize the known values with all fixed cells?

Reply:
Oh and there is https://github.com/powdr-labs/powdr/issues/2369 to fix this